plugins/node/opentelemetry-instrumentation-mongodb/src/instrumentation.ts

```diff
@@ -417,6 +419,32 @@ export class MongoDBInstrumentation extends InstrumentationBase<
     span.setAttribute(SemanticAttributes.DB_STATEMENT, JSON.stringify(query));
   }
 
+  /**
+   * Triggers the response hook in case it is defined.
+   * @param span The span to add the results to.
+   * @param config The MongoDB instrumentation config object
+   * @param result The command result
+   */
+  private _handleExecutionResult(
+    span: Span,
+    config: MongoDBInstrumentationConfig,
```

Comment:
This is a function on the `MongoDBInstrumentation` class, so it has access to the config from `this`. no need to pass it as a parameter to the function.

Reply:
i'd rather sending it this way since it is called inside a callback - see how it is called on line 468

Reply:
Why is calling it from a callback a problem?
Not that I care too much if it stays this way, but I think it should work with `this` as well, no?

Reply:
Removed the parameter, and using `getConfig()`